examples/va/nfv/hci-ovs-dpdk-sriov/values.yaml

```diff
@@ -0,0 +1,23 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values-post-ceph
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  nodeset:
+    services:
+      - install-certs
+      - ceph-client
+      - ovn
```

Comment:
these services you can remove in pre ceph nodeset values. looks like duplicated and overriddenin post ceph.

Reply:
Deployment is done in the following way:
- preceph: ovs-dpdk-sriov working deployment  (as in ovs-dpdk-sriov architecture)
- postceph: Add hci to a ovs-dpdk-sriov deployment (as in hci architecture)
I would keep duplicated services for the following reason:
- This architecture is a mix of ovs-dpdk-sriov and hci architectures, so for maintenance reasons I would keep services as in those architectures.
- Preceph generates an working ovs-dpdk-sriov. That deployment is updated with ceph and some services are executed again in order to update it